<launch>
	<param name="vehicle_file" type="string" value="$(find riptide_descriptions)/config/puddles.yaml" />
	<node pkg="riptide_hardware" type="coprocessor.py" name="coprocessor_driver" output="screen" />
</launch>
